test/system/272-system-connection.bats

```diff
@@ -180,4 +180,72 @@ $c2[ ]\+tcp://localhost:54321[ ]\+true" \
     run_podman system connection rm mysshconn
 }
 
+@test "podman-remote: non-default connection" {
+    # priority:
+    #   1. cli flags (--connection ,--url ,--context ,--host)
+    #   2. Env variables (CONTAINER_HOST and CONTAINER_CONNECTION)
+    #   3. ActiveService from containers.conf
+    #   4. RemoteURI
+
+    # setup
+    run_podman system connection add test1 unix:/run/user/test1/podman/podman.sock   # default=true
+    run_podman system connection add test2 unix:/run/user/test2/podman/podman.sock
+    run_podman system connection add test3 unix:/run/user/test3/podman/podman.sock
```

Comment:
I've tried but can't find a way to refactor the tests. However, I think they would be a lot more readable if you
1. rename test1-test2-test3 to "defaultconnection", "env-override", "cmdline-override" or something along those lines; and
2. remove the comments before each assertion, and edit the assertion messages instead.

I can't believe I said "remove comments".